package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@theia/monorepo",
   "version": "0.0.0",
   "engines": {
-    "node": ">=18"
+    "node": ">=22.12.0"
```

Comment:
What about node 20? For Browser it's still a thing.

Reply:
I'm fine either way, the only reason I set it to this value is because of this warning if you don't:
```
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'node-abi@4.9.0',
npm warn EBADENGINE   required: { node: '>=22.12.0' },
npm warn EBADENGINE   current: { node: 'v20.18.3', npm: '10.8.2' }
npm warn EBADENGINE }
```

Reply:
I adapted it to Node 20 now as I don't have any hard feelings about this